qt-models/filtermodels.cpp

```diff
@@ -75,7 +75,7 @@ bool MultiFilterSortModel::showDive(const struct dive *d) const
 		bool found_tag = false;
 		for (const auto& filter_tag : m_filterData.tags) {
 			for (const auto& dive_tag : dive_tags) {
-				if (filter_tag == dive_tag) {
+				if (filter_tag.trimmed() == dive_tag.trimmed()) {
```

Comment:
this code was introduced in the previous commit. Just fix the previous commit, please